airflow/providers/openlineage/provider.yaml

```diff
@@ -101,6 +101,15 @@ config:
         example: full.path.to.ExtractorClass;full.path.to.AnotherExtractorClass
         default: ~
         version_added: ~
+      custom_facet_functions:
+        description: |
+          Register functions that return custom facets to append to the run facet. This config takes a
+          string of semicolon delimited full import paths to the functions. The functions should accept
+          a TaskInstance input argument and should return a dictionary.
```

Comment:
Not sure if we should include that here as it's an implementation detail. If we change the input parameters it may be easy to forget to change that in the yaml, but maybe I'm wrong?
> The functions should accept a TaskInstance input argument and return a dictionary.